pandas/core/generic.py

```diff
@@ -4285,6 +4307,31 @@ def ftypes(self):
         """
```

Comment:
It would better organized it there was pull request for dtypes and another pull request for ftypes.